server/src/controllers/internship.ts

```diff
@@ -12,13 +12,15 @@ import {
   getUserWithInternshipModule,
 } from "../helpers/userHelper";
 import { buildHtmlTemplate, buildPDFFile, saveFile } from "../helpers/pdfHelper";
-import { User } from "../models/user";
+import { IUser, User } from "../models/user";
 import { constants } from "http2";
 import * as QueryString from "qs";
+import { ICompany } from "../models/company";
+import { collectInternships } from "./company";
 
-const INTERNSHIP_FIELDS_VISIBLE_FOR_USER =
+export const INTERNSHIP_FIELDS_VISIBLE_FOR_USER =
   "_id company tasks operationalArea programmingLanguages livingCosts salary paymentTypes status";
-const INTERNSHIP_FIELDS_ADDITIONALLY_VISIBLE_FOR_ADMIN =
+export const INTERNSHIP_FIELDS_ADDITIONALLY_VISIBLE_FOR_ADMIN =
   "startDate endDate workingHoursPerWeek supervisor";
```

Comment:
Ich würde vorschlagen, die beiden in eine Datei in utils auszulagern, damit die Controller keine Abhängigkeiten untereinander haben